Makefile

```diff
@@ -207,12 +192,10 @@ endef
 
 # Output names
 EXES        := \
-  $(sort $(filter $(BUILD),$(filter-out $(DONT_BUILD),$(PROGS))))
-
-INSTALL     ?= $(EXES)
+  $(sort $(filter $(UTILS),$(filter-out $(SKIP_UTILS),$(PROGS))))
 
 INSTALLEES  := \
-  $(sort $(filter $(INSTALL),$(filter-out $(DONT_INSTALL),$(EXES) uutils)))
+  $(sort $(filter $(UTILS),$(filter-out $(SKIP_UTILS),$(EXES) uutils)))
```

Comment:
On Mon, Jan 18, 2016 at 05:45:39PM -0800, Alex Lyon wrote:
> >  
> >  INSTALLEES  := \
> > -  $(sort $(filter $(INSTALL),$(filter-out $(DONT_INSTALL),$(EXES) uutils)))
> > +  $(sort $(filter $(UTILS),$(filter-out $(SKIP_UTILS),$(EXES) uutils)))
> 
> Isn't this the same as `INSTALLEES := $(EXES) uutils`?

It is. Updated.